packages/utilities/psammead-styles/src/global-styles.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-expressions */
+import styledNormalize from 'styled-normalize';
+import { createGlobalStyle } from 'styled-components';
+import { arrayOf, string } from 'prop-types';
+
+const createFontStyles = fonts =>
+  fonts.reduce((fontStyles, fontStyle) => fontStyles + fontStyle, '');
+
+const boxSizing = `/* Box Sizing https://bit.ly/1A91I0J */
+    html {
+      box-sizing: border-box;
+      font-size: 100%;
+    }
+    *, *:before, *:after {
+      box-sizing: inherit;
+    }`;
+
+const GlobalStyles = createGlobalStyle`
+    ${styledNormalize}
+    
+    ${boxSizing}
+
+    ${({ fonts }) => fonts && createFontStyles(fonts)}
```

Comment:
This creation of fonts in the `createFontStyles` method is very specific to our Simorgh application. It'd be better to decouple our implementation in Simorgh from this package, but retaining the ability to pass in fonts. 
```suggestion
    ${({ fonts }) => fonts ? fonts : null;}
```